packages/one-app-dev-bundler/utils/bundle-external-fallbacks.js

```diff
@@ -0,0 +1,61 @@
+import path from 'path';
+import { readPackageUpSync } from 'read-pkg-up';
+import esbuild from 'esbuild';
+import snakeCase from 'lodash.snakecase';
+import generateESBuildOptions from '../esbuild/generateESBuildOptions.js';
+import generateIntegrityManifest from './generate-integrity-manifest.js';
+
+const EXTERNAL_PREFIX = '__holocron_external';
+
+const { packageJson } = readPackageUpSync() || {};
+const { 'one-amex': { bundler = {} } } = packageJson;
+const { requiredExternals } = bundler;
+
+const getExternalLibraryName = (name, version) => [EXTERNAL_PREFIX, snakeCase(name), version.replace(/[^\d.]+/g, '').replace(/\.+/g, '_')].filter(Boolean).join('__');
+
+/**
+ * Transpiles Requires Externals as individual files.
+ * It's completely independent from bundling module's code.
+ */
+export const bundleExternalFallbacks = async () => {
+  if (
+    requiredExternals && Array.isArray(requiredExternals)
+    && requiredExternals.length > 0
+  ) {
+    const {
+      commonConfig,
+      browserConfig,
+    } = await generateESBuildOptions({ watch: false, useLiveReload: false });
+
+    const config = {
+      ...browserConfig,
+      plugins: [
```

Comment:
correct, but i'm overriding the plugins because the `browserConfig` provides plugins that are for modules only, for example, `externalsLoader`, `oneAppIndexLoader`, `globalExternals`, etc. Externals only need to be transpiled. There are some plugins that we could use from browser like `timeBuild`, and, since we are moving this config into `generateESBuildOptions` then it would be a good idea to add these type of plugins